src/main/java/Duke.java

```diff
@@ -1,25 +1,54 @@
 import java.util.Scanner;
 
 public class Duke {
-    public static void main(String[] args) {
 
-        boolean isByeEntered = true;
-        String outputs;
+    public static final int MAX_INT = 100;
 
+    public static void greeting() {
         System.out.println(" Hello! I'm Duke");
         System.out.println(" What can I do for you?");
 
-        while(isByeEntered) {
+    }
+
+
+
+    public static void add() {
```

Comment:
Coding Standard violation: Ambiguity of function.

Can change the name of the function to explain more. e.g. addTodo()